samples/boards/x86_info/prj.conf

```diff
@@ -0,0 +1,6 @@
+CONFIG_X86_ACPI=y
```

Comment:
samples/boards/ is for actual boards, this is not a board specific sample, need a better place, maybe we should make a test out of this instead?

Reply:
`tests/arch/x86/x86info`? I thought a sample was better because it's a utility rather more than a test, but I don't really care where it goes.

Reply:
yes, thats fine.

Reply:
moved to `tests/arch/x86/info`